spec/index.md

```diff
@@ -4546,22 +4556,52 @@ A record with
 
 -   `{certificate: C}`
 
-The predicate `may_read_path` is defined as follows, implementing the access control outlined in [Request: Read state](#http-read-state):
 
-    may_read_path(S, _, ["time"] · _) = True
-    may_read_path(S, _, ["subnet"] · _) = True
-    may_read_path(S, _, ["request_status", Rid] · _) =
+The predicate `may_read_path_for_canister` is defined as follows, implementing the access control outlined in [Request: Read state](#http-read-state):
+
+    may_read_path_for_canister(S, _, ["time"] · _) = False
+    may_read_path_for_canister(S, _, ["subnet"] · _) = False
```

Comment:
I believe we can actually drop these from here and allow implementations to serve these paths from both endpoints for backward compatibility. But I think it's also ok to have them be served by both endpoints. I'll go with whatever is the spec owner preference here.

Reply:
If we define `False` here, then we it shouldn't be permitted to serve those paths (what's the point of this predicate if it was permitted anyway?).

Reply:
Ok, maybe I wasn't clear on my intention. I think it's fine from a spec point of view that we drop these from here (and therefore they would be caught in the last predicate below which is the catch all). This would mean that these paths should not be permitted given a new version of the spec released with this change. However, the replica implementation should not immediately drop support for those, otherwise we would have a breaking change (which I would like to avoid).

Reply:
Discussed in the Interface spec meeting: we agreed to keep the `/time` and `/subnet` paths available via the `/api/v2/canister` URL, too (the `/time` is anyway always included and it makes sense to allow `/subnet` for backward-compatibility, too).